package.json

```diff
@@ -9,10 +9,11 @@
     "build": "gulp build",
     "lint": "gulp lint",
     "lint:sass": "gulp lint-sass",
-    "lint:pug": "gulp lint-pug"
+    "lint:pug": "gulp lint-pug",
+    "test": "npm run lint"
   },
   "pre-commit": [
-    "lint"
+    "test"
```

Comment:
Hey man since pre-commit hook takes an array, we can do something like this
```
  "lint": "gulp lint",
  "test": "mocha"
},
"pre-commit": [
  "test",
  "lint"
]
```

So it still can be pretty extensible :stuck_out_tongue: